test/expected/ddl_alter_column.out

```diff
@@ -61,7 +61,10 @@ SELECT * FROM alter_test WHERE time > '2017-05-20T10:00:01';
 
 -- rename column and change its type
 ALTER TABLE alter_test RENAME COLUMN time TO time_us;
+--converting timestamptz->timestamp should happen under UTC
+SET timezone = 'UTC';
```

Comment:
What does this mean? That it is only valid to convert timestamptz if TZ is UTC? Should we even allow such conversions in that case? Or, is this strictly something required only for our tests?

Reply:
Yeah for now I think it's ok to leave this as valid only under UTC. Didn't want to put in too much work here as this is a weird edge case but wanted to keep consistent for tests. I think it's ok not to block since you get a constraint violation error anyway if you do something wrong.

Reply:
Can't we just add a check in `process_utility` that the timezone is UTC and raise an error otherwise?